tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore2IastTests.cs

```diff
@@ -573,6 +573,8 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
     [Trait("RunOnWindows", "True")]
     public async Task TestIastStoredXssRequest()
     {
+        throw new SkipException("End point missing due to changes in Samples.Security.AspNetCore2 - refactoring in progess to restore");
```

Comment:
This is the same as doing this, right?

```csharp
[SkippableFact(Skip = "End point missing due to changes in Samples.Security.AspNetCore2 - refactoring in progess to restore")]
```

We _always_ want to skip, right? These tests only run on .NET Core 2.1?

Reply:
Yes, definitely only run on Core 2.1, we always want to skip because it can't work anywhere. I wasn't aware of the other way to skip test, I'll use that next time.

Reply:
Should be re-enabled soon anyway.